src/parameters/query/executionRequests/Status.yaml

```diff
@@ -0,0 +1,5 @@
+in: query
+name: status
+required: false
+schema:
+  type: string
```

Comment:
type should be enum. The options should be all of the different possible statusses: SUCCESS, WARNING, ERROR....